# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/audioipc/audioipc/src/messagestream_unix.rs
# Commit: 0a7873a6b522
# Full Hash: 0a7873a6b522ce4aef9a32ae4a47f093ba33e0f4
# Author: Matthew Gregan <kinetik@flim.org>
# Date: 2020-01-26 09:48:16
# Regressor Bug: 1590249
# File Overlap Count: 3
# Description:
#   Bug 1590249 - Update audioipc to 86d49ddf.  r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D60615
# ==============================================================================

diff -r 36f6505b50c9 -r 0a7873a6b522 media/audioipc/audioipc/src/messagestream_unix.rs
--- a/media/audioipc/audioipc/src/messagestream_unix.rs	Sat Jan 25 21:30:14 2020 +0000
+++ b/media/audioipc/audioipc/src/messagestream_unix.rs	Sat Jan 25 21:31:32 2020 +0000
@@ -39,6 +39,12 @@
     }
 }
 
+impl IntoRawFd for MessageStream {
+    fn into_raw_fd(self) -> RawFd {
+        self.0.into_raw_fd()
+    }
+}
+
 impl AsyncMessageStream {
     fn new(stream: tokio_uds::UnixStream) -> AsyncMessageStream {
         AsyncMessageStream(stream)
@@ -97,9 +103,3 @@
         self.0.as_raw_fd()
     }
 }
-
-impl IntoRawFd for MessageStream {
-    fn into_raw_fd(self) -> RawFd {
-        self.0.into_raw_fd()
-    }
-}